### DISCUSSION TITLE
How to integrate Tabby with GNS3?

### DISCUSSION THREAD

**Discussion by ed-dz** [Eugeny/tabby#9453] [General]
Hello folks, I'm trying to integrate Tabby with GNS3 (https://gns3.com/) but it keeps failing with different errors.  For example:

1. If I manually Telnet to one of my routers, the console just hangs. 
![image](https://github.com/Eugeny/tabby/assets/33724247/f2205100-fadc-4981-bc6c-149dadde6fe8)

2. If I try to setup GNS3 to us Tabby as the default terminal, Tabby did not open with the router session.
Basically, when using PuTTY as default terminal, it would open each router console when I double click from GSN3 lab, but creating a custom Terminal with Tabby (It doesn't exist from GNS3) the console goes straight to Windows CMD

![image](https://github.com/Eugeny/tabby/assets/33724247/e3769b51-f478-43c3-a20e-8c02b22f6a97)
![image](https://github.com/Eugeny/tabby/assets/33724247/facb8afc-7f90-490d-9749-a74df29d2767)

Does anybody knows how to fix this?
Thanks

**Comment by Blackers33** [comment]
Hello, what is working for me is :

`"C:\Program Files\Tabby\Tabby.exe" quickConnect telnet {host}:{port}`

I still haven't found a way to properly name the tab, it's showing the IP address at the moment.
